types/slate/slate-tests.ts

```diff
@@ -362,7 +358,6 @@ editor
     path: 'a',
     offset: 0,
     text: 'text',
-    marks: [Mark.create({type: 'test_mark'})],
```

Comment:
Why has this test case been removed? I don't see anything in the patch notes indicating this is no longer valid

Reply:
Looking at the diff of the commit it has been remove from the exposed method.
 
https://github.com/ianstormtaylor/slate/commit/a220cd5ae10c7e5f7df6d018deae0670526dbb73#diff-7ff73d3c1ad52a0d47563bac44a04389

Let me know if im missing something.

Reply:
@sgreav Have you had a chance to review my command, I would like to get this merged in ASAP. Cheers